mfthomps/CyberCIEGE-port-to-Unity3d
# issue/183-user-factory-avatar-prefab

Closes #183 
This refactors how the User prefabs get instantiated. Now, the UserFactory has a scriptable variable with the gender-to-gameObject mapping. Previously, the UserBehavior itself would switch on the correct child GameObject.

To verify:
- Run any scenario and verify Users (male and female workers) get created as before.
-
